cmd/deploy/deploy.go

```diff
@@ -303,61 +324,48 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 			SkipIfExists: !deployOptions.Dependencies,
 		}
 		if err := pipelineCMD.ExecuteDeployPipeline(ctx, pipOpts); err != nil {
+			if errStatus := updateConfigMapStatus(ctx, cfg, c, data, err); err != nil {
```

Comment:
Would the original execution error for the pipeline be lost if we return `errStatus`? Should we be using `wraps` What info does the user get about these errors? Do they get any info that a manifest is going to be created?

Reply:
Apart from Jacob's questions, I think this condition is not right, we should check `errStatus != nil` instead of `err != nil`.

I think we should not return `errStatus` that is for something "internal". We could not update the configmap which is the internal resource we use to handle deployments. The user should see the error about the pipeline execution